sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2052,6 +2029,33 @@ public partial class BackupPolicyMigrationState
         public static bool operator !=(Azure.ResourceManager.CosmosDB.Models.BackupPolicyType left, Azure.ResourceManager.CosmosDB.Models.BackupPolicyType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class BackupSchedule
```

Comment:
The name is too generic, consider to rename it with a more descriptive and meaningful name.

Reply:
Changed to CassandraClusterBackupSchedule